perun-core/src/main/java/cz/metacentrum/perun/core/entry/MembersManagerEntry.java

```diff
@@ -1434,20 +1434,25 @@ public RichMember sponsorMember(PerunSession session, Member sponsored, User spo
 
 	@Override
 	public List<RichMember> getSponsoredMembers(PerunSession sess, Vo vo) throws PrivilegeException, VoNotExistsException {
+		//Filter members based on authorization
```

Comment:
Please, use here the original politic `getSponsoredMembers_Vo_policy`. The GUI needs it to resolve authorization.

Reply:
Ok, fixed.

Reply:
I think the policy check is still missing.

Reply:
Sorry, I misunderstood, I hope it's fixed now.